lib/twingly/url.rb

```diff
@@ -103,11 +108,26 @@ def try_addressable_normalize(addressable_uri)
         raise
       end
 
+      def valid_hostname?(domain)
```

Comment:
nitpick: method implies we're validating a hostname but we pass a `domain` variable, ignore me If I don't get the details 👼

Reply:
Should be called `hostname` here (but we pass `public_suffix_domain.domain` but not necessary to reflect that here)